src/hotspot/os/linux/os_linux.hpp

```diff
@@ -193,6 +193,10 @@ class os::Linux {
 
   // none present
 
+  // Calls out to GNU extension malloc_info if available
+  // otherwise does nothing and returns 0.
+  static int malloc_info(FILE* stream);
+
```

Comment:
Could you put this at the end of the file, alongside os::get_mallinfo(), and within its ifdef GLIBC boundaries?